includes/embeds/class-amp-youtube-embed-handler.php

```diff
@@ -162,7 +223,7 @@ private function get_video_id_from_url( $url ) {
 		} else {
 			/* phpcs:ignore Squiz.PHP.CommentedOutCode.Found
 			The query looks like ?v={id} or ?list={id} */
-			parse_str( $parsed_url['query'], $query_args );
+			parse_str( $parsed_url['query'], $query_args ); // @todo Bug! See <https://github.com/ampproject/amp-wp/issues/3348>.
```

Comment:
To be done in this PR or another?

Reply:
That needs to be removed as the [`embed_oembed_html`](https://github.com/ampproject/amp-wp/pull/3358/commits/35260fd6b5068f75b987ffc631ab87366d9f24a9#diff-c0516153a81ae9941d43b32041e16347R53) filter correctly handles the URL and transforms it into an embeddable one.